doc/configuring.rst

```diff
@@ -389,34 +396,35 @@ Below is an example of the configuration file that uses :meth:`~Configuration.du
         print(configs.dump())
 
 The combination of ``#!/usr/bin/env {python}`` and ``if __name__ == '__main__':`` allows launching
-the `configs.py` script from shell.
+the `Universum` configuration files as a script from shell.
 
-For ``from universum.configuration_support import`` to work correctly, `configs.py` should be copied to
-`Universum` root directory and launched there.
+If `Universum` is not installed locally, for ``from universum.configuration_support import`` to work correctly
+the configuration file should be copied to local `Universum` root directory and launched there.
 
 When launched from shell instead of being used by `Universum` system, :ref:`get_project_root` function
 returns current directory instead of actual project root.
 
-The only thing this script will do is create `configs` variable and print all project configurations
-it includes. For example, running the script, given above, will result in the following:
+The only thing this script will do is create `configs` variable and print all build steps it includes.
+For example, running the script, given above, will result in the following:
 
 .. testcode::
     :hide:
 
-    print("$ ./configs.py")
+    print("$ ./.univesrum.py")
     print(configs.dump())
 
 .. testoutput::
 
-    $ ./configs.py
+    $ ./.univesrum.py
     [{'name': 'Make Special Module', 'command': 'make -C SpecialModule/', 'artifacts': 'out'},
     {'name': 'Run internal tests', 'command': 'scripts/run_tests.sh'},
     {'name': 'Run external tests', 'directory': '/home/scripts', 'command': 'run_tests.sh -d /home/Project/out/tests'}]
 
-As second and third build configurations have the same names, if log files are created,
-only two logs will be created: one for the first build step, another for both second and third,
-where the third will follow the second.
+As second and third steps have the same names, if log files are created, only two logs will be created:
```

Comment:
The second and third steps have different names.
Either the example or note has to be fixed.